Vagrantfile

```diff
@@ -32,6 +32,7 @@ module OS
 end
 
 required_plugins = %w(vagrant-triggers)
+required_plugins.push('vagrant-proxyconf')
```

Comment:
Only require this if `HTTP_PROXY` or `HTTPS_PROXY` are not empty. See below for `OS.windows?` block.